WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -523,6 +542,7 @@ private extension NotificationsViewController {
 
     func setupConstraints() {
         // Inline prompt is initially hidden!
+        tableHeaderView.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
Quick question: Why was this disabled? I couldn't see any change that warrants this, i.e. constraint additions to `tableHeaderView`.